obs_github_deployments.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "rubocop", "~> 1.7"
   spec.add_dependency "dry-cli", "~> 0.6"
+  spec.add_dependency "octokit", "~> 4.20"
```

Comment:
Instead of relying on the GitHub REST API, did you consider using the [GitHub GraphQL API](https://docs.github.com/en/graphql)? I believe it would be better since it is a lot more flexible in how we can fetch data.

Reply:
I totally agree with you @dmarcoux, GraphQL would be a cool thing. But I don't know if we have the time now to learn GraphQL. How about keeping it simple, so people can also start on other ends of the project soon, and then if time is left adapting the code to use GraphQL?

Reply:
I also think that we only need to do a few concrete calls to GitHub API, and they are really straightforward with `octokit`.
So at this point, I don't see the necessity to move to GraphQL.

Reply:
With the GraphQL API, we can directly fetch the state of the latest deployment. This isn't possible in the REST API.

It's pretty easy to experiment with the queries we need with GitHub's GraphQL explorer. Here's how we can do this. Keep in mind that we don't have any deployment now, so it's obviously returning nothing. 
![graphql-api-query-example](https://user-images.githubusercontent.com/1102934/108984917-bccf4300-7690-11eb-9c18-669f6d75f28b.png)

Reply:
We discussed about using the GraphQL API. We can start with the REST API version and if the GraphQL API version is satisfying, we'll switch to it.